recipes/azure-storage-cpplite/meta.yaml

```diff
@@ -0,0 +1,75 @@
+{% set name = "azure-storage-cpplite" %}
+{% set version = "0.3.0" %}
+
+package:
+  name: {{ name|lower }}
```

Comment:
This might need a different name from the packages below. Typically we go with something like `azure-storage-cpplite-split`.